pages/content/content1.js

```diff
@@ -0,0 +1,18 @@
+import Headline from '../../components/content-topic-one';
+import { ContentTopicStepOne } from '../../components/content-topic-one';
+import { ContentTopicStepTwo } from '../../components/content-topic-one';
+import Placeholderimg from '../../components/placeholderimg';
+
+function content1() {
```

Comment:
your pages should have a more descriptive name. What will be displayed on this page?

Reply:
Atm, the name is just a placeholder for later.